src/niscope/metadata/functions_addon.py

```diff
@@ -491,6 +491,16 @@
     'FancyFetch':                 { 'python_name': 'fetch', },
 }
 
+functions_parameter_names = {
```

Comment:
Most (all?) here are private. I think it's unnecessary to override, it's just more code to maintain. If you prefer to do it regardless for consistency that's ok but it should then be noted in a comment.

Reply:
They are and I didn't originally have them, but then there were things like `self._fetch_into_numpy(num_samples=num_samples, wfm=waveform, timeout=timeout)` so I went for consistency.

Comment added.